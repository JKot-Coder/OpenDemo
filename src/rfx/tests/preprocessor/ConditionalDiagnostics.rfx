// :RUN_RFXC -P %STD_OUTPUT% %INPUT% 
// Diagnostic Error Testing for 'Conditionals'

#else // directive without '#if'

#elif // directive after else 

// Testing EOF while conditional processing 
#ifdef UNDEFINED
Skip this
