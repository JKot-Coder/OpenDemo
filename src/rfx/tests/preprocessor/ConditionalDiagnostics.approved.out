exit_status:0
output:
Source output:
    #line 1 "D:/Work/my/OpenDemo/src/rfx/tests/preprocessor/ConditionalDiagnostics.rfx"
    
Diagnostic output:
    D:/Work/my/OpenDemo/src/rfx/tests/preprocessor/ConditionalDiagnostics.rfx:4:2: error: #else without #if
    #else // directive without '#if'
     ^
    D:/Work/my/OpenDemo/src/rfx/tests/preprocessor/ConditionalDiagnostics.rfx:6:2: error: #elif without #if
    #elif // directive after else 
     ^
    D:/Work/my/OpenDemo/src/rfx/tests/preprocessor/ConditionalDiagnostics.rfx:9:2: error: unterminated conditional directive
    #ifdef UNDEFINED
     ^
